### PR TITLE
Finished everything expect worship service

### PR DESCRIPTION
**Pull request by YJouhar47**

### CODE REVIEW

File: app/helpers/kbo-format.js [hunk 1]
```diff
@@ -1,20 +1,6 @@
...
-  }
-  const [one, two, three] = [
-    kbo.slice(0, 4),
-    kbo.slice(4, 7),
-    kbo.slice(7, 10),
-  ];
-  return `${one}.${two}.${three}`;
-}
+export default helper(function kboFormat([kbo]) {
+  if (!kbo || kbo.length !== 10) return '';
```
Comment (DennisVanEecke) [inline app/helpers/kbo-format.js:4]: Added errors (assertions) to the function. Don't just return empty string.
Check if parameter is string, if not error.
Check if length is ok, if not error. Even better. Use regex. `^\d{9}$`
You WANT the application to crash asap when something is wrong.
Especially helper functions for templates need type checks.

File: app/routes/core-data/admin-unit.js [hunk 2]
```diff
@@ -23,7 +25,7 @@ function assert(record, source, modelName) {
 export default class AdminUnitRoute extends Route {
   @service store;
   @service currentSession;
-
+  @tracked isCity = false;
```
Comment (DennisVanEecke) [inline app/routes/core-data/admin-unit.js:28]: What's the point of this? Makes no sense.

**Review by DennisVanEecke — CHANGES_REQUESTED** [review]
Please see comments. Mostly about kbo helper function. Not much work to fix.

**Review by DennisVanEecke — APPROVED** [review]
M'kay

**Review by andreo141 — APPROVED** [review]

**Pull request merged**